docs/patch.md

```diff
@@ -206,7 +147,7 @@ Current Value     | Given Value     | Result
 `null`            | `{a: 1}`        | `{a: 1}`
 `{}`              | `{a: 1}`        | `{a: 1}`
 `{a: 1}`          | `{b: 2}`        | `{a: 1, b: 2}`
-`{} `             | `{}`            | `{}`
+`{}`              | `{}`            | `{}`
```

Comment:
I've just noticed that this entry is duplicated. It is the same as the first one.